.env.mysql.dev

```diff
@@ -0,0 +1,12 @@
+DATABASE_USER=root
```

Comment:
I like the idea of having different env files for different DBs on dev environment, but don't like that it's in the root directory. Probably we can create a new `dev` directory to keep all of that stuff there?

Reply:
good suggestion on `dev` directory, I think this is a lot cleaner, I'm going to move the `.env.dev*` files there and considering also moving `DEVELOPER.md` there as well, thoughts?

Reply:
Makes sense, probably also can rename it to `README.md` so it shows up when you visit the directory on GH